contracts/GelatoRelayContextERC2771.sol

```diff
@@ -66,16 +67,40 @@ function _getMsgSenderRelayContextERC2771() pure returns (address _msgSender) {
  *     fee: - 52 bytes
  *     _msgSender: - 20 bytes
  */
+
+/**
+ * @dev If you are using in combination with ERC2771Context from OpenZeppellin
+ * i.e. you are wanting to use multiple relayers
+ * https://docs.openzeppelin.com/contracts/4.x/api/metatx
+ * You will have to set a `trustedForwarder` in your target contract's constructor:
+ * constructor(address _trustedForwarder) GelatoRelayContextERC2771(_trustedForwarder);
+ *
+ * You can set the `trustedForwarder` as:
+ * 1. A trusted relayer address other than GelatoRelayERC2771.sol
+ * 2. If no backup relayer is needed, and you are only using Gelato Relay:
+ *      a. set `trustedForwarder` to address(0)
+ * 3. Otherwise, DO NOT set the trustedForwarder as GelatoRealyERC2771
+ *      a. this is hard coded already, please use either:
+ *          I. onlyGelatoRelayERC2771()
+ *          II. _isGelatoRelayERC2771()
+ */
+
 /// @dev Do not use with GelatoRelayFeeCollectorERC2771 - pick only one
-abstract contract GelatoRelayContextERC2771 is GelatoRelayBase {
+abstract contract GelatoRelayContextERC2771 is
+    ERC2771Context,
```

Comment:
@sashaaldrick @jack-the-pug - we dont need to inherit the whole `ERC2771Context` code - we just want to inherit `Context` alone to achieve our compatibility and override aims of WP-S13